src/main/java/Event.java

```diff
@@ -0,0 +1,17 @@
+public class Event extends Task{
+    protected String date;
+    protected String description;
+
+    public Event(String description, String date){
+        this.description = description;
+        this.date = date;
+    }
+    @Override
+    public String getDescription(){
+        return this.description + "(" + this.date + ")";
```

Comment:
Avoid unnecessary use of this with fields. There are a few occurrences.
Refer to: https://se-education.org/guides/conventions/java/index.html#variables

Reply:
fixed thanks